### DISCUSSION TITLE
Typo in "Recent" tab

### DISCUSSION THREAD

**Discussion by tricantivu** [community/community#125713] [Mobile]
### Select Topic Area

Bug

### Body

Whenever I open the GitHub Mobile app, the "Recent" tab has the following typos:

![IMG_8515](https://github.com/community/community/assets/96702705/11354763-dd56-434c-9e92-c88175065b84)

It should say:

```md
Las propuestas y solicitudes de cambio con las que has interactuado recientemente aparecerán aquí
```

NOTE: I edited the suggestion

**Comment by taki-on** [comment]
@tricantivu Thank you for reporting this! I'd like to file a bug, but want to know if it happens on iOS or Android 🙇

**Reply by tricantivu** [reply]
> @tricantivu Thank you for reporting this! I'd like to file a bug, but want to know if it happens on iOS or Android 🙇 

iOS, my current version is  `17.4.1 (21E236)`.

Additionally, the GitHub mobile app's version is `1.162.0`

**Comment by Ryu141** [comment]
![Uploading Screenshot_2024-05-28-06-08-34-76.jpg…]()
